src/clarity-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -373,6 +373,11 @@
             vertical-align: bottom;
         }
 
+        .datagrid-signpost-trigger {
+            height: $clr_baselineRem_1_5 - 1;
+            padding: 0 12px 0;
+        }
```

Comment:
I'm not sure. 
It might be that the signpost is causing it to look like it works. 
I'll look at it again.